src/assets/package.json

```diff
@@ -4,25 +4,64 @@
   "license": "MIT",
   "scripts": {
     "deploy": "webpack --mode production",
-    "watch": "webpack --mode development --watch"
+    "watch": "webpack --mode development --watch",
+    "lint": "eslint --max-warnings 0 --ext .vue,.ts js",
+    "lint:fix": "eslint --fix --ext .vue,.ts js",
+    "test": "jest -u"
   },
   "dependencies": {
+    "axios": "^0.21.1",
+    "bootstrap-vue": "^2.21.2",
+    "buefy": "^0.9.4",
+    "eslint": "^7.22.0",
     "phoenix": "file:../deps/phoenix",
-    "phoenix_html": "file:../deps/phoenix_html"
+    "phoenix_html": "file:../deps/phoenix_html",
+    "vue": "^2.6.12",
+    "vue-router": "^3.5.1",
```

Comment:
Sorry if I'm wrong, you can remove `vue-router` because you don't use any `vue-router` function.

Reply:
haha yes nice catch, I forgot to delete it when decided not to use it